src/routes/AppRouter.js

```diff
@@ -1,28 +1,42 @@
 import React from 'react';
-import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';
-import {Products} from '../pages/Products/Products';
-import {Cart} from '../pages/Cart/Cart';
-import {Home} from '../pages/Home/Home';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { Products } from '../pages/Products/Products';
+import { Cart } from '../pages/Cart/Cart';
 import { Login } from '../pages/Login/Login';
-import {Header} from '../components/Header';
+import { Header } from '../components/Header';
+import { useSelector } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
 
 export const AppRouter = () => {
+  const isLoggedIn = useSelector((state) => state.login.isLogged)
+
   return (
     <Router>
       <Header />
       <Switch>
-        <Route exact path='/'>
-          <Home />
-        </Route>
-        <Route path='/products'>
-          <Products />
-        </Route>
-        <Route path='/cart'>
-          <Cart />
-        </Route>
-        <Route path='/login'>
-          <Login />
-        </Route>
+
+        {
+          isLoggedIn
+          ?
+          <>
+            <Redirect to='/products'/>
+            <Route path='/products'>
+              <Products />
+            </Route>
+            <Route path='/cart'>
+              <Cart />
+            </Route>
+          </>
+          :
+          <>
+            <Redirect to='/login'/>
+            <Route path='/login'>
+              <Login />
+            </Route>
+          </>
+        }
+
```

Comment:
I like that you do not allow it to go to the routes if the user is not authenticated